drivers/servo/servo.c

```diff
@@ -21,26 +23,80 @@
 #include "servo.h"
 #include "periph/pwm.h"
 
+#define ENABLE_DEBUG    (0)
+#include "debug.h"
+
+#define SEC_IN_USEC     (1000000U)
```

Comment:
I don't really mind. Just remembered that I've seen this "issue" commented in some PRs. Let's see what Hauke says.